src/actions/postOnInstagram.ts

```diff
@@ -0,0 +1,17 @@
+import { withActionOutput } from '../utils';
+
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Opens the photo passed into this action in the Instagram app and copies the caption to the clipboard.
+ *
+ * ```js
+ * postOnInstagram();
+ * ```
+ */
+const postOnInstagram = (): WFWorkflowAction => ({
+  WFWorkflowActionIdentifier: 'com.burbn.instagram.openin',
+  WFWorkflowActionParameters: {},
```

Comment:
Could you add the `caption` param into this please, and add a test for it too?

Reply:
Added